src/ConnectedMode/Migration/IMigrationPrompt.cs

```diff
@@ -35,7 +36,7 @@ namespace SonarLint.VisualStudio.ConnectedMode.Migration
     /// </summary>
     internal interface IMigrationPrompt : IDisposable
     {
-        Task ShowAsync(BoundSonarQubeProject oldBinding);
+        Task ShowAsync(BoundSonarQubeProject oldBinding, bool isAlreadyConnected);
```

Comment:
Probably worth adding a doc comment to this interface now.

Reply:
Could we call `isAlreadyConnected` something else to avoid confusion? (since we use it somewhere else to mean something subtly different)

e.g. `hasNewBinding`, `hasNewBindingFiles` ?

Reply:
Hmm what do you mean we use it to mean something different? Does it not mean the exact same?

Reply:
Here it means "has connected mode settings".

In the other usage it means "SonarQubeService.Connected=true". 

They are subtly different.